src/com/amazon/ion/impl/lite/IonValueLite.java

```diff
@@ -76,6 +80,9 @@ abstract class IonValueLite
     private   static final int ELEMENT_MASK       = 0xff;
     protected static final int ELEMENT_SHIFT      = 8; // low 8 bits is flag, upper 24 (or 48 is element id)
 
+    // This value was chosen somewhat arbitrarily; it can/should be changed if it is found to be insufficient.
+    private static final int CONTAINER_STACK_INITIAL_CAPACITY = 16;
```

Comment:
We have a similar variable in a few places now. Is it worth consolidating them at some point?

Reply:
Alternatively, why not go with the default value which also happens to be 16 (I'm assuming that's why 16 was chosen).

Reply:
The default ArrayDeque initial size happens to be 16, but I don't want my container stack initial capacity to be whatever the default ArrayDeque capacity is in JDK 8 or JDK 45. I want it to be 16 because deeper nesting than that in Ion data is assumed to be exceedingly rare.